projects/step-frontend/src/lib/modules/plan/components/plan-artefact-list/plan-artefact-list.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, EventEmitter, Inject, Input, Output } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { downgradeComponent, getAngularJSGlobal } from '@angular/upgrade/static';
+import { AJS_LOCATION, AJS_MODULE, AugmentedControllerService } from '@exense/step-core';
+import { ILocationService } from 'angular';
+import { ArtefactService } from '../../services/artefact.service';
+
+@Component({
+  selector: 'step-plan-artefact-list',
+  templateUrl: './plan-artefact-list.component.html',
+  styleUrls: ['./plan-artefact-list.component.scss'],
+})
+export class PlanArtefactListComponent {
+  @Output() onSelection = new EventEmitter<string>();
+
+  constructor(
+    readonly _artefactService: ArtefactService,
+    readonly _controllerApiService: AugmentedControllerService,
```

Comment:
Can not find the usage of this service or it's datasource.
Is it required? If not I think both field and augmented service, can be removed